### DISCUSSION TITLE
Do I need to be on HA OS 2023.x.x for Spaclient 2.6b to work?

### DISCUSSION THREAD

**Discussion by Jieper001** [plmilord/Hass.io-custom-component-spaclient#30] [Q&A]
Major noob here.  I am currently still on HA OS 2022.9.7 and run the Spaclient v2.5.  Absolutely love what you have created as I now can automate the heating behaviour on my Spa.  Love it!

However, in preparing to migrate my HA to a bigger SSD, I just upgraded to Spaclient v2.6b whilst still on HA OS 2022.9.7.

After restart I got the message that "Integration not loaded".  And none of the Spaclients' entities were available.

Freaked out and reverted immediately back to my full back, and now all is again good.

My question: do I need to upgrade to HA OS >= 2023.x.x for Spaclient v2.6b to work?

Thank you - very grateful!!
J

**Comment by plmilord** [comment]
As we have removed the remaining use of the deprecated temperature utility by HA... Yes, you may need to update to the latest version for this to work...
